src/translations/en.json

```diff
@@ -51,8 +52,12 @@
     "insertNewDirectoryDialogMessage": "Please insert the following information! ",
     "insertNewDirectoryDialogTitle": "Create folder",
     "insertNewRootDirectoryDialogTitle": "Create root folder",
-    "deleteDirectoryDialogMessage": "Are you sure you want to delete this folder?",
-    "deleteDirectoryDialogTitle": "Delete the folder",
+    "deleteDirectoryDialogMessage": "Are you sure you want to delete the {directoryName} folder and the items it contains?",
+    "deleteItemDialogMessage": "Are you sure you want to delete {elementName} ?",
+    "deleteMultipleItemsDialogMessage": "Are you sure you want to delete those {itemsCount} items ?",
+    "deleteDirectoryDialogTitle": "Delete {directoryName} folder",
+    "deleteItemDialogTitle": "Delete item {elementName}",
+    "deleteMultipleItemsDialogTitle": "Delete {itemsCount} items",
```

Comment:
Should see plural management in internationalization to limit translations entries
see deleteElementsFailure for exemple